test.py

```diff
@@ -0,0 +1,94 @@
+#fasta
+fasta = open("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta", "r")
+
+seq = fasta.read()
```

Comment:
`open`は単体で使っちゃだめ。`open`した時はファイルを独占的に使用していることになるから、`read`で使用し終わったらすぐに`close`をつけないといけない。
ただ、`close`をつけるのは人為的ミスしやすいから、`with`構文を使おう
https://qiita.com/1023shogo2013/items/1e03965c89efe63ba66d
```suggestion
with open("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta", "r") as fasta:
    seq = fasta.read()
```